_pages/infrastructure/aws.md

```diff
@@ -78,21 +78,22 @@ At a minimum, an AWS resource must have a `Project` tag defined with enough info
 
 ## Creating new accounts
 
-Someone from the Tech Portfolio should do the following:
-
-1. Email Tara Hopkins at [thopkins@jhctechnology.com](mailto:thopkins@jhctechnology.com) asking for a new AWS sub-account
-   - Customer Name (Agency, Dept, etc.)
-   - Project/Program Name
-   - Optional Preferred Full Name (format: `<CUSTOMER> - <PROJECT>`): `tts-<program>-<env>`
-   - Optional Preferred Account Login email: `devops+aws-<program>-<env>@gsa.gov`
-   - Lead Technical Contact
-     - Name
-     - Email
-     - Phone
-   - Note if you need an associated GovCloud account
-1. JHC will send account ID once created
-1. Add to [account list](https://docs.google.com/spreadsheets/d/1DedSCiU9AsCAAVvAFZT0_Ii7AFIKlI-JNifzlpHNbDg/edit#gid=0)
-1. Reset password by email
+1. Forecast the spending for the next 6-12 months.
+   - If you expect the spend across your accounts to increase by more than a few percent, the contract may need to be mofified.
+1. Reach out to the [Tech Portfolio](https://handbook.18f.gov/tech-portfolio/) with info about:
```

Comment:
Where should they reach out to the Tech Portfolio? Based on this, they will likely reach out on the [#Infrastructure](https://gsa-tts.slack.com/messages/infrastructure/)slack channel since that is listed on that page, but I think it seems like they should be directing these requests to [#aws-admin](https://gsa-tts.slack.com/messages/admins-iaas/) instead.